common/origin/origin.go

```diff
@@ -0,0 +1,87 @@
+package origin
+
+import (
+	"github.com/opencontrol/fedramp-templater/common/source"
+	"gopkg.in/fatih/set.v0"
+	"log"
+	"reflect"
+	"strings"
+)
+
+// Key is a unique value that represents the different types of possible control origination value.
+type Key uint8
+
+// Origination prefixes.
+const (
+	NoOrigin Key = iota
+	ServiceProviderCorporateOrigination
+	ServiceProviderSystemSpecificOrigination
+	ServiceProviderHybridOrigination
+	ConfiguredByCustomerOrigination
+	ProvidedByCustomerOrigination
+	SharedOrigination
+	InheritedOrigination
+)
+
+// SrcMapping is a data structure that represents the text for a particular control origination in a particular source.
+type SrcMapping map[source.Source]string
+
+// IsDocMappingASubstrOf is wrapper that checks if the input string contains the SSP mapping.
+// This is useful because the input string value may have extra characters so we can't do a == (equal to) check.
```

Comment:
Whitespace, or more than that? If the former, we should just do a [`strings.Trim()`](https://golang.org/pkg/strings/#Trim) on the input.